build/ci.go

```diff
@@ -523,7 +523,7 @@ func doDebianSource(cmdline []string) {
 				build.MustRunCommand("debsign", changes)
 			}
 			if *upload != "" {
-				build.MustRunCommand("dput", *upload, changes)
+				build.MustRunCommand("dput", "--passive", "--no-upload-log", *upload, changes)
```

Comment:
The `--passive` is deprecated though, so might not always be there. I guess we don't have a profile file? Might be better to use `-override "passive_ftp=true"`, so we don't have to get bit when they suddenly remove it in a couple of years